packages/scm/src/browser/scm-widget.tsx

```diff
@@ -100,16 +94,8 @@ export class ScmWidget extends ReactWidget implements StatefulWidget {
         this.toDisposeOnRefresh.dispose();
         this.toDispose.push(this.toDisposeOnRefresh);
         const repository = this.scmService.selectedRepository;
-        this.title.label = ScmWidget.LABEL;
-        if (repository) {
-            this.title.label += ': ' + repository.provider.label;
-        }
-        const area = this.shell.getAreaFor(this);
-        if (area === 'left') {
-            this.shell.leftPanelHandler.refresh();
-        } else if (area === 'right') {
-            this.shell.rightPanelHandler.refresh();
-        }
+        this.title.label = repository ? repository.provider.label : 'no repository opened';
```

Comment:
![Screen Shot 2019-08-01 at 15 25 43](https://user-images.githubusercontent.com/914497/62297082-cbcc2000-b470-11e9-9644-57431abe53ce.png)

cannot see the repository opened anymore.

![Screen Shot 2019-08-01 at 15 27 28](https://user-images.githubusercontent.com/914497/62297111-e3a3a400-b470-11e9-9ead-3e4402ab3a85.png)

this is in vscode: `B` is the repo name on FS ;-)

Reply:
Our scm widget behaves different to scm widget from VS Code for multiple repos:
- in VS Code you have a new part for each repo, so they have to disambiguate them with a label
- in Theia there is always only one part, and a user see a repo and can change in the status bar

I would say you should file an issue if you want to align it with VS Code, although i don't see how their approach is better? It introduces custom handling for a widget instead of relying on the same code.

Reply:
Thanks for explaining!

> in Theia there is always only one part, and a user see a repo and can change in the status bar

I like the fact that it's in the status bar a lot, and coercing multiple instances of the scm widget into that area could be less useful.